GameData/RealismOverhaul/RO_RecommendedMods/TACLS/RO_TACLS_Tanks.cfg

```diff
@@ -220,10 +377,37 @@
 	@description = A life support recycling module that uses the Sabatier reaction to reclaim water from carbon dioxide and hydrogen. Produces 2kg LqdMethane/day.
 	@MODULE[TacGenericConverter]
 	{
+		//conversionRate = 2.0	// Amount LqdMethane/Day
```

Comment:
Ah ok. I removed it initially. But it is back in in the latest version/s.
I changed all the rates to apply the coversion rate in this PR.
Could change them back again as conversionRate is now available again, but would have to un-apply the conversion rate to the individual resource values.